test/factories.ts

```diff
@@ -86,7 +86,13 @@ export const radioFactory = fieldFactoryForType('radio')
 export const optionSelectFactory = fieldFactoryForType('optionSelect')
   .attrs({ options: attrOptions });
 
+export const objectSearchFactory = fieldFactoryForType('objectSearch')
+  .attrs({
+    endpoint: '/endpoint/',
+  });
+
 export const objectSearchCreateFactory = fieldFactoryForType('objectSearchCreate')
+  .extend(objectSearchFactory)
   .attrs({
     createFields: [{ field: 'name', required: true }],
     endpoint: '/endpoint/',
```

Comment:
If you do end up extending `objectSearchFactory`, this attribute is redundant

Reply:
I suppose so?